reports/inventory/helpers.ts

```diff
@@ -17,6 +17,7 @@ export async function getRawStockLedgerEntries(fyo: Fyo) {
     'name',
     'date',
     'item',
+    "batchNumber",
```

Comment:
@akshayitzme check if `prettier` is working, the codebase uses single quotes. The rules are mentioned in package.json.

Just a heads up, ensure that your code is formatted according to the codebase before you push it.

Reply:
that slipped my mind. thanks, I will ensure it from now on